source/Octopus.Tentacle.Tests/Client/RpcCallRetryHandlerFixture.cs

```diff
@@ -473,11 +469,12 @@ public async Task CancelsTheExecutingActionIfItIsARetryAfterTheTimeoutEvenIfActi
             }
             catch (HalibutClientException) { }
 
-            stopWatch.Elapsed.Should().BeGreaterOrEqualTo(TimeSpan.FromSeconds(8)).And.BeLessThan(TimeSpan.FromSeconds(20));
+            stopWatch.Elapsed.Should()
+                .BeGreaterOrEqualTo(handler.RetryTimeout - handler.RetryIfRemainingDurationAtLeast - retryBackoffBuffer)
```

Comment:
This could sometimes be slightly less than 8 seconds. Use a more "calculated" was of determining the allowed time